CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 2.2.2 (Unreleased)
+## 2.3.0 (Unreleased)
+
+ENHANCEMENTS:
+
+* `resource/repository`:  Added functionality to generate a new repository from a Template Repository ([#309](https://github.com/terraform-providers/terraform-provider-github/issues/309))
```

Comment:
Nitpicks: There's double space and the reference here could be just `[GH-309]` and our release script will turn it into a link automatically as part of the release.

Reply:
I've addressed these nits.  Much appreciated.